Fixes #311 Support reload language model without restart of IDEA

mydeveloperplanet opened the pull request:
Tested it with LMStudio and works like a charm :-)

mydeveloperplanet commented:
chose for the refresh-button solution
![image](https://github.com/user-attachments/assets/7b70fb65-66e1-4b4c-9181-8fe1221032f4)

stephanj commented:
Make sure the button uses the full available height.

![image](https://github.com/user-attachments/assets/2b3ff8ac-383e-46ff-b72b-57141b1ef897)

mydeveloperplanet commented:
Fixed the button size and refactored some code. Works even better now :-) 
![image](https://github.com/user-attachments/assets/2d1208a9-45cb-4e63-aae3-e5407e68eeee)
